.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,7 +6,7 @@
 
 #### Prerequisites checklist
 
-- [ ] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).
+- [ ] I have read the [contributing guidelines](../CONTRIBUTING.md).
```

Comment:
Using a relative link here will make the link in the last generated PR content inaccessible.

Reply:
Good point.  Fixed.